commands/hugo.go

```diff
@@ -127,6 +127,10 @@ func initializeConfig(mustHaveConfigFile, failOnInitErr, running bool,
 		return nil, err
 	}
 
+	for _, s := range c.hugo().Sites {
```

Comment:
I think we have some commands that needs to complete even without a valid config, meaning `c.hugo()` will just block forever. It would ease my mind if you rewrote the above to:

```go
if h := c.hugoTry(); h != nil {
range h.Sites ...
}
...